lib/cisco_node_utils/overlay_global.rb

```diff
@@ -131,7 +131,9 @@ def anycast_gateway_mac
     def anycast_gateway_mac=(mac_addr)
       fail TypeError unless mac_addr.is_a?(String)
 
-      Feature.nv_overlay_evpn_enable
+      Feature.nv_overlay_evpn_enable unless Feature.nv_overlay_evpn_enabled?
```

Comment:
You don't need the `unless` checks since the Feature setters already check for that.